Doc/library/enum.rst

```diff
@@ -573,6 +589,20 @@ Data Types
       better support the *replacement of existing constants* use-case.
       :meth:`__format__` was already :func:`int.__format__` for that same reason.
 
+.. class:: ReprEnum
+
+   :class:`!ReprEum` uses the :meth:`repr() <Enum.__repr__>` of :class:`Enum`,
+   but the :class:`str() <str>` of the mixed-in data type:
+
+      * :meth:`int.__str__` for :class:`IntEnum` and :class:`IntFlag`
+      * :meth:`str.__str__` for :class:`StrEnum`
+
+   Inherit from :class:`!ReprEnum` when mixing in a data type when the
+   :class:`str() <str>` output should not be changed to the
```

Comment:
```suggestion
   Inherit from :class:`!ReprEnum` to keep the :class:`str() <str> / :func:`format`
   of the mixed-in data type instead of using the
```

After giving this a closer look, I found my original confusing and think it reads much clearer this way (and avoids a double "when")